src/components/modals/SearchModal/SearchBox.tsx

```
@@ -0,0 +1,113 @@
+                    {item.price} $
+                  </p>
+                  {item.discountPercentage && (
+                    <BlackBadge className='w-12'>
+                      -{item.discountPercentage}%
+                    </BlackBadge>
+                  )}
+                </div>
+              </div>
+            </div>
+          </li>
+        ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)